code/__DEFINES/is_helpers.dm

```diff
@@ -228,3 +228,7 @@ GLOBAL_LIST_INIT(book_types, typecacheof(list(
 	/obj/item/book,
 	/obj/item/spellbook,
 	/obj/item/storage/book)))
+
+//Datums
+
+#define is_reagent_container(O) (istype(O, /obj/item/reagent_containers))
```

Comment:
Could you put this one along with the other item defines?

Reply:
I'm honestly unsure how I looked at /obj/ and read /datum/. I think I have datum mind.